src/engine/video/gl/gl_render_target.cpp

```diff
@@ -0,0 +1,292 @@
+////////////////////////////////////////////////////////////////////////////////
+//            Copyright (C) 2012-2015 by Bertram (Valyria Tear)
+//                         All Rights Reserved
+//
+// This code is licensed under the GNU GPL version 2. It is free software
+// and you may modify it and/or redistribute it under the terms of this license.
+// See http://www.gnu.org/copyleft/gpl.html for details.
+////////////////////////////////////////////////////////////////////////////////
+
+/** ****************************************************************************
+*** \file    gl_render_target.cpp
+*** \author  Authenticate, James Lammlein
+*** \brief   Source file for buffers for a render target.
+*** ***************************************************************************/
+
+#include "utils/utils_pch.h"
+#include "gl_render_target.h"
+
+#include "utils/exception.h"
+#include "utils/utils_strings.h"
+
+namespace vt_video
+{
+namespace gl
+{
+
+RenderTarget::RenderTarget(unsigned width,
+                           unsigned height) :
+    _width(width),
+    _height(height),
+    _framebuffer(0),
+    _texture(0),
+    _renderbuffer_depth(0)
+{
+    assert(_width > 0);
+    assert(_height > 0);
+
+    bool errors = false;
```

Comment:
The `errors` member and all the logic around it looks weird to me.
IMHO, as soon as you've got a `glGetError() != GL_NO_ERROR` you should print an error and trigger the end of app before returning if quit app is non-blocking and doesn't actual suffice to stop the game.

IMHO, all in all, you can remove all the `if (errors)` as they are redundant.
WDYT?